submissions/karelskiy/HTML and CSS practice/style.css

```diff
@@ -1,94 +1,106 @@
- body{
+body{
     margin: 0;
     font-family: 'Raleway', sans-serif;
 }
 
-.section{
-    position: absolute;
-    right:50px;
-    top: 3px;
+nav{
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    margin: 5px;
 }
-.menu-btn{
 
-    display: block;
-    width: 50px;
-    height: 50px;
-    background-color: #fff;
-    border-radius: 50%;
-    position: relative;
-    
+nav a.li, .menu-btn{
+    margin-left: 20px;
 }
-.menu-btn span,
-.menu-btn span::before,
-.menu-btn span::after {
-    position: absolute;
-    top: 50%;
-    margin-top : -1px;
-    left: 50%;
-    margin-left: -10px;
-    width: 20px;
-    height: 2px;
-    background-color: gray;
-}
-
-.menu-btn span::before,
-.menu-btn span::after{
-    content: '';
-    
+
+a.lImg, a.lMail, .fig a{
```

Comment:
Don't use too much-specific selectors. Try to use as simplest selector as it needs.